charts/kubefed/charts/controllermanager/templates/deployments.yaml

```diff
@@ -20,9 +20,7 @@ spec:
         runAsUser: 1001
       serviceAccountName: kubefed-controller
       containers:
-      - args:
-        - --kubefed-namespace=$(KUBEFED_NAMESPACE)
```

Comment:
Please remove the now-unused env entry for KUBEFED_NAMESPACE (L39-L43).

Reply:
Fixed